src/jace/__init__.py

```diff
@@ -5,13 +5,35 @@
 #
 # SPDX-License-Identifier: BSD-3-Clause
 
-"""
-JaCe: JAX jit using DaCe (Data Centric Parallel Programming)
-"""
+"""JaCe - JAX Just-In-Time compilation using DaCe."""
 
 from __future__ import annotations
 
+import jax as _jax
 
-__version__ = "0.1.0"
+import jace.translator.primitive_translators as _  # noqa: F401  # Populate the internal registry.
 
-__all__ = ["__version__"]
+from .__about__ import __author__, __copyright__, __license__, __version__, __version_info__
+from .jax import grad, jacfwd, jacrev, jit
+
+
+# In Jax `float32` is the main datatype, and they go to great lengths to avoid
+#  some aggressive [type promotion](https://jax.readthedocs.io/en/latest/type_promotion.html).
+#  However, in this case we will have problems when we call the SDFG, for some reasons
+#  `CompiledSDFG` does not work in that case correctly, thus we enable it now globally.
+_jax.config.update("jax_enable_x64", True)
```

Comment:
You are right it is not the best place.
The reason is that by default Jax traces everything as float32, even if the input is float64 unless this option is in place.
This then leads to errors if it is called in DaCe, Jax solves this issue by performing some hidden casting step.
However, currently this setting is essentially needed, but I fully agree with our point, so I moved it to where the actual tracing happens, further it will not affect the outside.